runtime/Cargo.toml

```diff
@@ -99,6 +102,7 @@ std = [
 	"frame-system-benchmarking?/std",
 	"frame-system-rpc-runtime-api/std",
 	"frame-system/std",
+	"pallet-proxy/std",
```

Comment:
why is it only added to `std` features? it also has `runtime-benchmarks` and `try-runtime` features. 
I don't know if all of them are required for this case, but just wanted to double check you haven't missed them